src/chapters/ldap-auth.xml

```diff
@@ -384,6 +384,17 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                                 have no effect on Guacamole configurations.</emphasis></para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-dereference-aliases</property></term>
+                    <listitem>
+                        <para>Controls whether or not the LDAP connection follows (dereferences) aliases
+                            as it searches the tree.  There are four possible values: never (the default),
```

Comment:
The values themselves need to be in quotes, so it's clear that their literal values must be specified verbatim. This is the convention that we follow elsewhere. For example, a few sections above:

    <para>The encryption mechanism that Guacamole should use when communicating
        with your LDAP server. Legal values are "none" for unencrypted LDAP,
        "ssl" for LDAP over SSL/TLS (commonly known as LDAPS), or "starttls" for
        STARTTLS. If omitted, encryption will not be used.</para>